meson_options.txt

```diff
@@ -2,7 +2,7 @@ option('build', type : 'combo', choices : ['all', 'standalone', 'library'], valu
 option('consolekit', type : 'feature', description : 'ConsoleKit support', deprecated: {'true': 'enabled', 'false': 'disabled'})
 option('static_analysis', type : 'boolean', value : false, description : 'enable GCC static analysis support')
 option('firmware-packager', type : 'boolean', value : true, description : 'enable firmware-packager installation')
-option('docs', type : 'combo', choices : ['none', 'gtkdoc', 'docgen'], value : 'docgen', description : 'developer documentation type')
+option('docs', type : 'boolean', value : true, description : 'Build developer documentation', deprecated: {'docgen': 'true', 'none': 'false'})
```

Comment:
Do you want to keep it enabled by default? I would argue it's not needed for most people since we publish to a URI anyway.

Or in a related thought should it be a meson tristate feature instead? Then it wouldn't be a hard dep by default.

Reply:
I don't think it wants to be a tristate-- it's not really an extra nice to have feature, it seems like something most people want to do; we're only syncing the docs at release time, and it might be the check-ed out copy is the most up to date docs available. That said, most people building a plugin a probably using other plugins and perhaps devhelp as well, rather than pointing firefox at the local docs. My gut instinct is it's an explicit boolean, but I could argue the default either way.

Reply:
Well as another thought... We could have two copies of docs published to the web. Latest stable release and also "tip" from CI.

Reply:
Yes, sane. Not for this PR tho :)

Reply:
Okay.. I say what you have is good for now and if/when we publish multiple docs copies it should flip default to off.